.scripts/util.sh

```diff
@@ -59,25 +63,73 @@ is_generated() {
     fi
 }
 
+select_browser() {
+    local browser_engine="$1"
+
+    if [ "$browser_engine" = "Chromium" ]; then
+        BROWSER="Chromium"
+        BROWSER_INCLUDE_DIR="$CHROMIUM_INCLUDE_DIR"
+        BROWSER_LIB_DIR="$CHROMIUM_LIB_DIR"
+        SYS_OPTS="-DCHROMIUM_INCLUDE_DIR=$BROWSER_INCLUDE_DIR -DCHROMIUM_LIB_DIR=$BROWSER_LIB_DIR"
+    elif [ "$browser_engine" = "MSWebView2" ]; then
+        BROWSER="MSWebView2"
+        BROWSER_INCLUDE_DIR="$MSWEBVIEW_INCLUDE_DIR"
+        BROWSER_LIB_DIR="$MSWEBVIEW_LIB_DIR"
+        SYS_OPTS="-DMSWEBVIEW_INCLUDE_DIR=$BROWSER_INCLUDE_DIR -DMSWEBVIEW_LIB_DIR=$BROWSER_LIB_DIR"
+    else
+        throw_error "Unsupported browser engine option: $browser_engine. Program will terminate."
+        return 1
+    fi
+}
+is_browser_selected() {
+    if [ -z "$BROWSER" ]; then
+        throw_error "A browser must be selected. Program will terminate."
+        return 1
+    fi
+    return 0
+}
+
+install_cef() {
+    local nuget=$PACKAGE_DIR/bin/nuget.exe
+    mkdir -p "$PACKAGE_DIR/bin"
+
```

Comment:
Keep it [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) by moving this to a seperate function call, eg `verify_nuget(){}` which will install it if not installed, or simply call it once within the top level `packages_install` function with a guard for Windows.